src/cc/arduino/plugins/wifi101/UpdaterImpl.java

```diff
@@ -31,17 +31,26 @@
 import java.util.List;
 
 import javax.swing.JOptionPane;
-
+import javax.swing.DefaultListModel;
 import cc.arduino.packages.BoardPort;
-import cc.arduino.plugins.wifi101.firmwares.WINC1500Firmware;
 import cc.arduino.plugins.wifi101.flashers.Flasher;
-import cc.arduino.plugins.wifi101.flashers.java.JavaFlasher;
+import cc.arduino.plugins.wifi101.flashers.java.WINCFlasher;
+import cc.arduino.plugins.wifi101.flashers.java.NinaFlasher;
 import processing.app.Base;
 
 @SuppressWarnings("serial")
 public class UpdaterImpl extends UpdaterJFrame {
-
 	private Flasher flasher;
+  private SerialPortListModel listModel;
+	private List<String> websites = new ArrayList<>();
+
+	public static Flasher fwAvailable[] = new Flasher[] {
+			new NinaFlasher ("WifiNINA firmware ", "1.0.0", "firmwares/WifiNINA/1.0.0/nina-fw.bin"),
```

Comment:
Here, NINA is a better folder name. Is the contents of the 1.0.0 binary that same as what's used in the factory?